iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChanger.java

```diff
@@ -50,6 +51,8 @@
      */
     C setTapPosition(int tapPosition);
 
+    C removeTapPosition();
```

Comment:
There is a linked between this change and the subject of this PR? I am not sure to understand.

Reply:
Yes, the issue is that by making tap position optional, if we do not have this method, it means that once tap position is set, a user can never go back. Not sure if this is useful but that behavior seemed weird to me.

Reply:
Javadoc is missing

Reply:
I would call it unsetTapPostion, I think "remove" is not correct